src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -536,6 +538,17 @@ VirtualRepeatController.prototype.link_ =
 };
 
 
+/** @private Cleans up all created blocks (used and unused). */
+VirtualRepeatController.prototype.cleanupBlocks_ = function() {
+  function cleanupBlock(block) {
+    block.element.remove();
+  }
+
+  angular.forEach(this.blocks, cleanupBlock);
```

Comment:
This one is probably not needed (they should be removed along with the parent element). Doesn't hurt much to leave it in, though.